cat-launcher/src-tauri/src/infra/utils.rs

```diff
@@ -29,7 +29,17 @@ pub async fn read_from_file<T: DeserializeOwned>(path: &Path) -> Result<T, ReadF
     Ok(v)
 }
 
-#[derive(Debug, PartialEq)]
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+use strum::{Display, EnumString};
+use ts_rs::TS;
+
+#[derive(
```

Comment:
Why are these extra attributes required? Remove all which are not required, including the strum and serde directives.